Overview.bs

```diff
@@ -1610,12 +1610,11 @@ The algorithm:
 
 <dfn abstract-op>Handle errors</dfn>
 
-Coming soon.
-
-<!-- TODO -->
-
-<!-- TODO: we should consider defining a valid encoded IFT font as one that produces a correct font subset for the corresponding
-           definition in the entry. This would likely be the only requirement made for encoders. -->
+If the extending the font subset process has failed with an error, the client can continue to use the current extended font subset
+(from prior to the error occuring) to render the content, while falling back to the another font to render parts of the content not
```

Comment:
I don't disagree with any of this but it seems tricky to get right in practice so I'm wondering if this is a bit understated. Are you thinking a client might make a copy with a modified cmap? Or maybe (in effect) put the right combination of unicode-range notations in place around it?

And is there a definitive means of determining what code points the initial file or a given patch level supports? (I suppose there probably is, I'm just worried about the "can-provide-more-than-what-was requested" cases.)

What if the failure had was trying to add an axis, or range of an axis? What if it was adding a feature?

Reply:
Yeah you'e right, I wasn't considering the case where the cmap overdeclares coverage that is expected to be filled in later via patches. I can see two approaches that we could take for the error handling:

1. Recommend a font that has suffered a failure during extension not be used at all, this is the safest most conservative option.
2. Allow the font to be used after failure but only for codepoints that are listed in the cmap and are not listed in any of the patch entries that are not yet loaded (ie. so any codepoints that we are reasonably confident do not have any pending loads). There of course could still be missing functionality w/ features and or design space, we could add some additional text cautioning against using a partially loaded font in those cases.

What do you think?

Reply:
Either of these options seem OK to me. i suppose the decision should rest on whether the cost of communicating (and comprehending) the more liberal option is outweighed by the benefit. I doubt it would be in the short term because it would be annoying to implement correctly, but specifications aren't just for the short term.

Reply:
I took a pass at rewriting this. I decided to present both options and leave it up to the client to decide which to use. I can see both being valid options for a client depending on the specifics of the client.